src/bin/pgcopydb/defaults.h

```diff
@@ -6,6 +6,21 @@
 #ifndef DEFAULTS_H
 #define DEFAULTS_H
 
+/*
+ * Setup Boehm-Demers-Weiser conservative garbage collector as a garbage
+ * collecting replacement for malloc. See https://www.hboehm.info/gc/
+ */
+#include <gc/gc.h>
+
+#define malloc(n) GC_malloc(n)
+#define calloc(m, n) GC_malloc((m) * (n))
+#define free(p) GC_free(p)
+#define realloc(p, n) GC_realloc((p), (n))
+#define CHECK_LEAKS() GC_gcollect()
```

Comment:
It comes from there: https://www.hboehm.info/gc/leak.html ; I'm thinking it might be good to keep it around in case we want to do memory analysis later on, but we could also remove it.